packages/sources/spectral-macro-score/schemas/env.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$id": "https://external-adapters.chainlinklabs.com/schemas/example-source-adapter.json",
+  "title": "@chainlink/example-source-adapter env var schema",
+  "required": [],
+  "type": "object",
+  "properties": {},
```

Comment:
Need to update id, title, required and properties. Here's an example: https://github.com/smartcontractkit/external-adapters-js/blob/develop/packages/sources/json-rpc/schemas/env.json